package.json

```diff
@@ -32,5 +32,8 @@
     "babel-eslint": "^10.1.0",
     "husky": "^8.0.3",
     "standard": "^17.1.0"
+  },
+  "dependencies": {
+    "lodash.isequal": "^4.5.0"
```

Comment:
We will need the `Npm.require` in `package.js` or https://atmospherejs.com/tmeasday/check-npm-versions
here to make this work properly. Probably the first one.

Reply:
I have now replaced require() in subscription.js with normal import syntax.